extensions/amp-a4a/0.1/head-validation.js

```diff
@@ -76,7 +76,7 @@ const EXTENSION_ALLOWLIST = map({
 });
 
 const EXTENSION_URL_PREFIX = new RegExp(
-  urls.cdn.replace(/\./g, '\\.') + '/v0/'
+  '^' + urls.cdn.replace(/\./g, '\\.') + '/(rtv/\\d*/)?v0/'
```

Comment:
1. `\d+` rather than `\d*`
2. At this point, does it make more sense to test `str.startsWith(prefix)`?

Reply:
Talked offline, we lose some strictness by moving to `startsWith` so kept regex with escape function from mdn. Changes to `+`.